core/src/test/scala/org/apache/spark/util/UtilsSuite.scala

```diff
@@ -915,6 +918,7 @@ class UtilsSuite extends SparkFunSuite with ResetSystemProperties with Logging {
   }
 
   test("Set Spark CallerContext") {
+    SparkHadoopUtil.get.conf.setBoolean("hadoop.caller.context.enabled", true);
```

Comment:
@srowen 
This test can also catch this case. But I think there is issue here, `"hadoop.caller.context.enabled"` always `false` for this test, so `if (CallerContext.callerContextSupported) ` always `false`. 
In addition, `CallerContext` added after hadoop-2.8.0 for HDFS-9184,  so we need set hadoop-version >= 2.8.0 in pom.xml. And then this test will throw exception  if I am not change `build` to `build: Class[_]`

Reply:
It sounds like tests using Hadoop 2.8+ would catch this case already. We do have a Hadoop 3.2 test job, so I wonder why it didn't trigger it. Nevertheless, I don't think you can force this to true here, yes. Maybe it's hard to test independently at all and so this can be removed.

Reply:
ok, it will be remove.

Reply:
> It sounds like tests using Hadoop 2.8+ would catch this case already. We do have a Hadoop 3.2 test job, so I wonder why it didn't trigger it. Nevertheless, I don't think you can force this to true here, yes. Maybe it's hard to test independently at all and so this can be removed.

ok, it will be remove.